deploy/helm-charts/values.yaml

```
@@ -0,0 +1,27 @@
+terrascan_api_key: terrakey
+terrascan_container_image: accurics/terrascan:1.5.0
+use_debug: true
+secret_name: terrascancerts
+config_name: terrascanconfig
+name: terrahook
```

```suggestion
name: terrascan
```
terrahook is a fun name but applicable only if we use the webhook `mode = true`
 .. better to go with the generic `terrascan` now

---

fixed